tst/regression/utils/test_case.py

```diff
@@ -207,20 +208,21 @@ def Run(self):
         print(run_command)
         sys.stdout.flush()
         try:
-            subprocess.check_call(run_command)
+            proc = subprocess.run(run_command, check=True, stdout=PIPE, stderr=PIPE)
+            return proc.stdout
         except subprocess.CalledProcessError as err:
             raise TestManagerError('\nReturn code {0} from command \'{1}\''
                               .format(err.returncode, ' '.join(err.cmd)))
 
-    def Analyse(self):
+    def Analyse(self, stdouts):
 
         test_pass = False
 
         print("*****************************************************************")
         print("Analysing Driver Output")
         print("*****************************************************************")
         sys.stdout.flush()
-        test_pass = self.test_case.Analyse(self.parameters)
+        test_pass = self.test_case.Analyse(self.parameters, stdouts)
```

Comment:
Follow instructions above and remove stdouts argument, it should be available in self.parameters.